plugins/gateway/kong/deploy/deploy.sh

```diff
@@ -4,13 +4,13 @@ KongGatewayDeploymentName="${KONG_GATEWAY_DEPLOYMENT_NAME:-kong}"
 KongGatewayDeploymentNamespace="${KONG_GATEWAY_DEPLOYMENT_NAMESPACE:-default}"
 KongGatewayIngressName="${KONG_GATEWAY_INGRESS_NAME:-demo}"
 KongGatewayIngressNamespace="${KONG_GATEWAY_INGRESS_NAMESPACE:-default}"
-UpstreamTelemetryHostName="${UPSTREAM_TELEMETRY_HOST_NAME:-apiclarity-apiclarity.apiclarity:9000}"
-TraceSamplingHostName="${TRACE_SAMPLING_HOST_NAME:-apiclarity-apiclarity.apiclarity:9990}"
+UpstreamTelemetryAddress="${UPSTREAM_TELEMETRY_ADDRESS:-apiclarity-apiclarity.apiclarity:9000}"
+TraceSamplingAddress="${TRACE_SAMPLING_ADDRESS:-apiclarity-apiclarity.apiclarity:9990}"
 TraceSamplingEnabled="${TRACE_SAMPLING_ENABLED:-false}"
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-cat "${DIR}/kongPlugin.yaml" | sed "s/{{UPSTREAM_TELEMETRY_HOST_NAME}}/$UpstreamTelemetryHostName/g" | sed "s/{{TRACE_SAMPLING_HOST_NAME}}/$TraceSamplingHostName/g" | sed "s/{{TRACE_SAMPLING_ENABLED}}/$TraceSamplingEnabled/g" | kubectl -n ${KongGatewayIngressNamespace} apply -f -
+cat "${DIR}/kongPlugin.yaml" | sed "s/{{UPSTREAM_TELEMETRY_ADDRESS}}/$UpstreamTelemetryAddress/g" | sed "s/{{TRACE_SAMPLING_ADDRESS}}/$TraceSamplingAddress/g" | sed "s/{{TRACE_SAMPLING_ENABLED}}/$TraceSamplingEnabled/g" | kubectl -n ${KongGatewayIngressNamespace} apply -f -
```

Comment:
For fun ;)
```shell
sed 's/{{UPSTREAM_TELEMETRY_ADDRESS}}/$UpstreamTelemetryAddress/g; s/{{TRACE_SAMPLING_ADDRESS}}/$TraceSamplingAddress/g; s/{{TRACE_SAMPLING_ENABLED}}/$TraceSamplingEnabled/g' ${DIR}/kongPlugin.yaml | kubectl -n ${KongGatewayIngressNamespace} apply -f -
```

Reply:
Looks really fun I admit, but what is not broken, do not fix :)